### PR TITLE
seed product data and create home page

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/76014

## WHAT (optional)
- Implement home page

## HOW
- Seed product data and display on home page

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/services/product.service.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      const products = await this.productRepository.createQueryBuilder('products')
+      .leftJoinAndSelect('products.categories', 'categories')
+      .where('categories.id = :categoryId', {categoryId: category.id}).getMany();
+      const item = {
+        category: category.name,
+        products: products,
+      };
+      result.push(item);
+    }
+    console.log('products: ', result);
```
Comment (huongnt-2545) [inline src/services/product.service.ts]: xóa debug

File: src/views/user/home.ejs [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+              <button class="btn btn-lg" type="button"><img src="/images/search-icon.png" class="search-icon" alt="search icon"></button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <h3 class="mb-4">
+        <%= t('home.newProduct') %>
+      </h3>
+      <div class="row g-4">
+        <% for(let i=0; i < newProduct.length; i++) { %>
```
Comment (huongnt-2545) [inline src/views/user/home.ejs]: vòng loop này dùng `forEach` đc ko e?

Comment (ghost) [reply src/views/user/home.ejs]: Dạ được ạ

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**